TheraLang.Web/ClientApp/src/app/core/auth/user.service.ts

```diff
@@ -55,4 +55,16 @@ export class UserService {
       responseType: "text"
     });
   }
+  getCurrentUserId()
+  {
+    let token: string = localStorage.getItem("jwt");
+    if (token && !this.jwtHelper.isTokenExpired(token)) {
+      let jwtData = this.jwtHelper.decodeToken(token);
+      let userId = jwtData['Id']
+      return userId;
+    }
+    else
+    return console.error("method: getCurrentUserId(), error: user is not authorized ");
```

Comment:
Returning 'console.error' wher user id expected is a bad idea, even worse than loggin frontend error

Reply:
return value changed to undefined